src/addons/metaservice/src/main/java/ee/ria/xroad/proxy/serverproxy/MetadataServiceHandlerImpl.java

```diff
@@ -148,17 +148,16 @@ public boolean canHandle(ServiceId requestServiceId,
 
         requestMessage = requestProxyMessage.getSoap();
 
-        switch (requestServiceId.getServiceCode()) {
-            case LIST_METHODS: // $FALL-THROUGH$
-            case ALLOWED_METHODS: // $FALL-THROUGH$
-            case GET_WSDL:
+        return switch (requestServiceId.getServiceCode()) { // $FALL-THROUGH$
+            // $FALL-THROUGH$
```

Comment:
are these double $FALL-THROUGH$ comments still needed here? Since it looks very specific, is it some type of hint to some code analyzer or something?

Reply:
looks like it doesn't do anything so removed